bayes/src/main/java/com/enigmastation/ml/bayes/Feature.java

```diff
@@ -16,12 +16,21 @@
 
 package com.enigmastation.ml.bayes;
 
+import java.io.IOException;
 import java.io.Serializable;
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * TODO: Needs to be done
+ */
 public class Feature implements Serializable {
-    Object feature;
```

Comment:
This is gross. It's a feature. It's not a featureObject. Java is strongly typed. If Sonar was complaining about this, it's complaining about too much.